OPUNetTransportLayer.cpp

```diff
@@ -222,13 +219,15 @@ bool OPUNetTransportLayer::GetExternalAddress()
 	packet.tlMessage.requestExternalAddress.internalPort = GetPort();
 
 	// Get the game server address
+	sockaddr_in gameServerAddr;
 	if (!GetGameServerAddress(gameServerAddr))
 	{
 		// Error. Could not obtain game server address
 		return false;		// Error
 	}
 
 	// Send the request packet to the game server (first port)
+	bool errorCode;
 	errorCode = SendTo(packet, gameServerAddr);

```

Comment:
Maybe simplify to say:

```suggestion
	bool errorCode = SendTo(packet, gameServerAddr);
```

and remove line 230

Reply:
Yes, you're right. Good eye for noticing that detail.

I was at first going to merge those two lines, until I noticed the second identical `SendTo` call on line 237. Merging would have made the first call visually different from the second identical call. You wouldn't have seen that in the diff due to the short code segments displayed.
https://github.com/OutpostUniverse/NetFixClient/blob/2d51333d3634024475996d89b0a75c63fc2a2c00/OPUNetTransportLayer.cpp#L229-L237

Other options, which would preserve the visual similarity, might have been to declare a second variable for the second call, or wrap both in independent code blocks so they could use the same variable name, declared twice.

At any rate, I expect to change that section of code in future edits in a way that will eliminate this. Essentially I've deferred on the issue.